.vscode/launch.json

```diff
@@ -17,7 +17,7 @@
             "name": "Debug Debugger",
             "type": "node",
             "request": "launch",
-            "program": "${workspaceRoot}/out/debugger/nodeDebugWrapper.js",
+            "program": "${workspaceRoot}/src/debugger/reactNativeDebugEntryPoint.ts",
```

Comment:
So, actually the goal of this change was to use `reactNativeDebugEntryPoint` rather than `nodeDebugWrapper` to run and attach debugger to, js -> ts is not necessary, so i can switch this back to using generated js.

Also this doesn't directly related to the bug this PR fixes so i'm not sure if this should be included in this PR. Pls. let me know if i need to submit a separate PR with this change.